### PR TITLE
Add audio gathering utility

### PR DESCRIPTION
**Pull request by daemon**

### CODE REVIEW

**Review by tuzhucheng — COMMENTED** [review]
For the Python projects we generally follow the [PEP 8](https://www.python.org/dev/peps/pep-0008/) style, so please use 4 spaces for indent.

Please also show a snippet of a sample command to run to record audio in the README.md.

Great work!

**Comment by daemon** [comment]
Okay, I've conformed to PEP 8 and written some info in the readme.

File: utils/manage_audio.py [hunk 1]
```diff
@@ -0,0 +1,203 @@
+import argparse
+import numpy as np
+import os
+import pyaudio
```
Comment (tuzhucheng) [inline utils/manage_audio.py:4]: Please create `environment.yml` (conda) or `requirements.txt` (pip) to list all dependencies for easy installation. Feel free to do this for the next PR.

**Review by tuzhucheng — APPROVED** [review]

**Comment by lintool** [comment]
msg to test Slack integration.

**Pull request merged**